LDKNodeMonday/Utilities/Constants.swift

```diff
@@ -99,3 +114,8 @@ struct Constants {
     }
 
 }
+
+struct EsploraServer: Hashable {
+    var name: String
+    var url: String
+}
```

Comment:
I like it, thanks for adding-

Reply:
Improved this by moving the static definitions into this struct, which also helps fix the ?? instead of ! issues in OnboardingViewModel